styles/tables/mod.css

```diff
@@ -1,44 +1,59 @@
+/* Modernized table styles */
+
 table {
-  display: block;
+  width: 100%;
   max-width: 100%;
   margin: 0 auto 1rem;
   border-collapse: collapse;
-  border-spacing: 0;
-  inline-size: fit-content;
   overflow-x: auto;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
 }
 
 .table-responsive {
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+  display: block;
+  width: 100%;
   overflow-x: auto;
 }
 
-.table-responsive table {
-  display: table;
-}
-
 caption {
-  margin-top: .5rem;
+  margin-top: 0.5rem;
   caption-side: bottom;
   color: var(--muted);
+  font-size: 0.875rem;
 }
 
 tbody > tr:nth-child(2n) {
   background: var(--bg-muted);
 }
 
 th, td {
-  padding: 0.375rem 0.8125rem;
+  padding: 0.75rem 1rem;
   border: 1px solid var(--bd-muted);
+  text-align: left;
 }
 
 th {
-  border-color: var(--bg-contrast);
   background: var(--bg-subtle);
-  font-weight: bold;
+  font-weight: 600;
+  color: var(--primary-color);
```

Comment:
The `--primary-color` doesn't exist.
Also removing the borders on table headers make the table quite difficult to read (especially on `colspan`):
![image](https://github.com/lowlighter/matcha/assets/22963968/a84f30f8-d004-4a07-9972-2cf58a48ec15)

Reducing bolding, uppercasing with letter-spacing seems fine though.